### PR TITLE
629 : Corrections in documentaion

### PR DESCRIPTION
**Pull request by Saket2**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Appsmith safely encrypts all your database credentials and stores them securely.
 
 ## What if I get stuck building an app on Appsmith?
 
-We sincerely hope this never happens to anyone but we acknowledge that there can be road blocks. At this juncture we're working on a way for you to be able to write custom react components and plugins to unblock you from any platform limitations. In the meantime, if absence of any feature is limiting you, you can write to [support@appsmith.com](mailto:%20support@appsmith.com) or reach out to us on our [discord server](https://discord.com/invite/rBTTVJp) 
+We sincerely hope this never happens to anyone but we acknowledge that there can be road blocks. At this juncture, we're working on a way for you to be able to write custom react components and plugins to unblock you from any platform limitations. In the meantime, if an absence of any feature is limiting you, you can write to [support@appsmith.com](mailto:%20support@appsmith.com) or reach out to us on our [discord server](https://discord.com/invite/rBTTVJp)
```
Comment (Nikhil-Nandagopal) [inline faq.md:25]: should be if "the" absence

File: widget-reference/datepicker.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 description: >-
-  Datepicker is used to capture the date and time from a user. It can be used to
-  filter data base on the input date range as well as to capture personal
+  Datepicker is used to capture the date and time of a user. It can be used to
```
Comment (Nikhil-Nandagopal) [inline widget-reference/datepicker.md:3]: change to "input by a user"

File: widget-reference/datepicker.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+  filter database on the input date range as well as to capture personal
```
Comment (Nikhil-Nandagopal) [inline widget-reference/datepicker.md:4]: database is incorrect here let's change to
filter data based on the input date range

File: widget-reference/dropdown.md [hunk 4]
```diff
@@ -40,9 +40,9 @@ Read more about submitting Input data to an API below
 | Property | Description |
 | :--- | :--- |
 | **Label** | Sets the label of the dropdown. |
-| **Selection Type** | Lets you control whether you want the dropdown to be a Single Select or a Multi Select dropdown. Single Select allows users to choose only one value from the list of options whereas Multi Select allows for selection of more than one value by the user. |
+| **Selection Type** | Lets you control whether you want the dropdown to be a Single Select or a Multi-Select dropdown. Single Select allows users to choose only one value from the list of options whereas Multi-Select allows for selection of more than one value by the user. |
```
Comment (Nikhil-Nandagopal) [inline widget-reference/dropdown.md:43]: multi select is sometimes starting with caps and sometimes not. Lets standardise it

File: widget-reference/filepicker.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ You can upload files by creating a post API and referring to the base64 version
       </td>
     </tr>
     <tr>
-      <td style="text-align:left"><b>uploadedFileUrls</b>
+      <td style="text-align:left"><b>uploadedFileURLs</b>
```
Comment (Nikhil-Nandagopal) [inline widget-reference/filepicker.md:39]: This is the name of the property in the codebase, we can't update it without changing the property

File: widget-reference/filepicker.md [hunk 6]
```diff
@@ -52,7 +52,7 @@ You can upload files by creating a post API and referring to the base64 version
 | **Allowed File Types** | Enables you to set constraints on the type of file a user can upload - images, videos, audios, text files, etc.  |
 | **Required** | When turned on, it makes a user input required and disables any form submission until an input is made. |
 | **Visible** | Controls widget's visibility on the page. When turned off, the widget will not be visible when the app is published  |
-| **Uploaded File URLs** | This property specifies the urls of the uploaded files that the file upload API returned. You can bind it to the API using {{ apiName.data.fileUrl }} |
+| **Uploaded File URLs** | This property specifies the URLs of the uploaded files that the file upload API returned. You can bind it to the API using {{ apiName.data.fileURL }} |
```
Comment (Nikhil-Nandagopal) [inline widget-reference/filepicker.md:55]: cannot change the name of a variable in the codebase

File: widget-reference/input.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ description: >-
 
 Input widgets can be used to allow users to search through records. The text of the input can be accessed in an API by referencing the text property **`{{ InputName.text }}`**
 
-The same can be used to filter data on the client side by writing a filter function using the text property
+Same can be used to filter data on the client-side by writing a filter function using the text property
```
Comment (Nikhil-Nandagopal) [inline widget-reference/input.md:14]: The same is correct in this case

**Comment by Saket2** [comment]
@Nikhil-Nandagopal - Made the suggested changes.

**Review by Nikhil-Nandagopal — APPROVED** [review]
@Saket2 thank you for your contribution 🎉 
This looks good so I'm merging it in

**Pull request merged**